tests/run-make/long-linker-command-lines/foo.rs

```diff
@@ -57,16 +47,14 @@ fn main() {
     let me_as_linker = format!("linker={}", env::current_exe().unwrap().display());
```

Comment:
Suggestion: slightly above this, panic if `RUSTC` env var is unavailable, instead of `unwrap_or`.